android/src/main/java/com/bitmovin/player/reactnative/converter/JsonConverter.kt

```diff
@@ -96,9 +98,67 @@ class JsonConverter {
                     playerConfig.adaptationConfig = it
                 }
             }
+            if (json.hasKey("remoteControlConfig")) {
+                toRemoteControlConfig(json.getMap("remoteControlConfig"))?.let {
+                    playerConfig.remoteControlConfig = it
+                }
+            }
             return playerConfig
         }
 
+        /**
+         * Converts an arbitrary [ReadableMap] to a [RemoteControlConfig].
+         *
+         * @param json JS object representing the [RemoteControlConfig].
+         * @return The generated [RemoteControlConfig].
+         */
+        private fun toRemoteControlConfig(json: ReadableMap?): RemoteControlConfig? {
+            if (json == null) return null
+            val defaultRemoteControlConfig = RemoteControlConfig()
```

Comment:
I was already thinking of this pattern in the past. I was a bit reluctant to actually do it, but it probably is a good option for the use case we have here.

Reply:
I took the inspiration from one of our discussions earlier. This is the best (only?) pattern to take the default values defined in the native code as well as be prepared for a potential immutable configuration object in the future.

Reply:
I agree.